### DISCUSSION TITLE
Is there a story to the name?

### DISCUSSION THREAD

**Discussion by EdorianDark** [BurntSushi/jiff#69] [Q&A] [answered]
For libraries like chrono and time it is clear how the authors got the name.  Is there a clever reason for the name?

**Comment by BurntSushi** [comment, marked as answer]
Does [this](https://docs.rs/jiff/latest/jiff/_documentation/design/index.html#why-name-the-library-jiff) answer your question?

**Reply by EdorianDark** [reply]
Thanks, I didn' see that.
